Content.Client/Lobby/LobbyState.cs

```diff
@@ -222,37 +225,52 @@ private void UpdateLobbyUi()
-
-            foreach (var session in _playerManager.Sessions.OrderBy(s => s.Name))
+            if (_gameTicker.LobbyBackground != null)
             {
-                var readyState = string.Empty;
-                // Don't show ready state if we're ingame
-                if (!gameTicker.IsGameStarted)
-                {
-                    LobbyPlayerStatus status;
-                    if (session.UserId == _playerManager.LocalPlayer?.UserId)
-                        status = gameTicker.AreWeReady ? LobbyPlayerStatus.Ready : LobbyPlayerStatus.NotReady;
-                    else
-                        gameTicker.Status.TryGetValue(session.UserId, out status);
-
-                    readyState = status switch
-                    {
-                        LobbyPlayerStatus.NotReady => Loc.GetString("lobby-state-player-status-not-ready"),
-                        LobbyPlayerStatus.Ready => Loc.GetString("lobby-state-player-status-ready"),
-                        LobbyPlayerStatus.Observer => Loc.GetString("lobby-state-player-status-observer"),
-                        _ => string.Empty,
-                    };
-                }
-
-                _lobby.OnlinePlayerList.AddItem(session.Name, readyState);
+                _lobby.Background.Texture = _resourceCache.GetResource<TextureResource>(_gameTicker.LobbyBackground );
             }
+            else
+            {
+                _lobby.Background.Texture = null;
+            }
+
         }
 
+
+        //private void UpdatePlayerList()
+        //{
+        //    if (_lobby == null) return;
+        //    _lobby.OnlinePlayerList.Clear();
+        //    var gameTicker = EntitySystem.Get<ClientGameTicker>();
+
+        //    foreach (var session in _playerManager.Sessions.OrderBy(s => s.Name))
+        //    {
+        //        var readyState = string.Empty;
+        //        // Don't show ready state if we're ingame
+        //        if (!gameTicker.IsGameStarted)
+        //        {
+        //            LobbyPlayerStatus status;
+        //            if (session.UserId == _playerManager.LocalPlayer?.UserId)
+        //                status = gameTicker.AreWeReady ? LobbyPlayerStatus.Ready : LobbyPlayerStatus.NotReady;
+        //            else
+        //                gameTicker.Status.TryGetValue(session.UserId, out status);
+
+        //            readyState = status switch
+        //            {
+        //                LobbyPlayerStatus.NotReady => Loc.GetString("lobby-state-player-status-not-ready"),
+        //                LobbyPlayerStatus.Ready => Loc.GetString("lobby-state-player-status-ready"),
+        //                LobbyPlayerStatus.Observer => Loc.GetString("lobby-state-player-status-observer"),
+        //                _ => string.Empty,
+        //            };
+        //        }
+
+        //        _lobby.OnlinePlayerList.AddItem(session.Name, readyState);
+        //    }
+        //}
```

Comment:
Is the plan add the player list back in? Otherwise why is all the code here and throughout `LobbyState.cs` just commented out?

Reply:
Woops totally forgot to clean that up, good catch. No plans to re add the lobby playerlist, I was worried about dependencies so i commented it out for testing and forgot about it